### ISSUE TITLE
Vercel source map upload failing with API Request Failed

### ISSUE THREAD

**Issue by russelh15** [getsentry/onpremise#491]
I've been trying to get source map uploading working with Vercel and our on premise Sentry setup.

I'm able to successfully log errors to Sentry, but I can't read the errors because of the source maps.

The uploading fails with a really generic error and I don't know how to proceed. I've Googled around but found nothing.

Anyone experience anything similar? I'm using NextJS hosted on Vercel (Now.sh/Zeit)

Log below. You'll notice it says API request failed, but does not provide why.

```
14:01:17.838  
  DEBUG   2020-05-11 18:01:17.838122082 +00:00 > Host: https://----:9000
14:01:17.838  
  DEBUG   2020-05-11 18:01:17.838130275 +00:00 > Accept: */*
14:01:17.838  
  DEBUG   2020-05-11 18:01:17.838137829 +00:00 > Connection: TE
14:01:17.838  
  DEBUG   2020-05-11 18:01:17.838145601 +00:00 > TE: gzip
14:01:17.838  
  DEBUG   2020-05-11 18:01:17.838153393 +00:00 > User-Agent: sentry-cli/1.52.3
14:01:17.838  
  DEBUG   2020-05-11 18:01:17.838178558 +00:00 > Authorization: Bearer ******
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175137398 +00:00 < HTTP/1.1 200 OK
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175174029 +00:00 < Access-Control-Allow-Headers: X-Sentry-Auth, X-Requested-With, Origin, Accept, Content-Type, Authentication, Authorization, Content-Encoding
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175184543 +00:00 < Access-Control-Allow-Methods: GET, POST, HEAD, OPTIONS
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175194535 +00:00 < Access-Control-Allow-Origin: *
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175204509 +00:00 < Access-Control-Expose-Headers: X-Sentry-Error, Retry-After
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175213578 +00:00 < Allow: GET, POST, HEAD, OPTIONS
14:01:18.175  
  DEBUG   2020-05-11 18:01:18.175222479 +00:00 < Content-Language: en
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175232761 +00:00 < Content-Type: application/json
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175241890 +00:00 < Vary: Accept-Language, Cookie
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175250561 +00:00 < X-Content-Type-Options: nosniff
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175258694 +00:00 < X-Frame-Options: deny
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175267561 +00:00 < X-XSS-Protection: 1; mode=block
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175276641 +00:00 < Content-Length: 297
14:01:18.176  
  DEBUG   2020-05-11 18:01:18.175286075 +00:00 < Connection: keep-alive
14:01:18.177  
  DEBUG   2020-05-11 18:01:18.176880111 +00:00 response status: 200
14:01:18.177  
  DEBUG   2020-05-11 18:01:18.176906836 +00:00 body: {"maxFileSize":2147483648,"hashAlgorithm":"sha1","concurrency":8,"chunkSize":8388608,"maxRequestSize":33554432,"url":"http://-----:9000/api/0/organizations/shiphero/chunk-upload/","chunksPerRequest":64,"accept":["debug_files","release_files","pdbs","sources"],"compression":["gzip"]}
14:01:22.514  
> Bundled 511 files for upload
14:01:23.751  
  DEBUG   2020-05-11 18:01:23.751674964 +00:00 error: running update nagger
14:01:23.752  
  DEBUG   2020-05-11 18:01:23.751708441 +00:00 skipping update nagger because session is not attended
14:01:23.752  
error: API request failed
14:01:23.752  
  caused by: [55] Failed sending data to the peer
14:01:23.752  
  DEBUG   2020-05-11 18:01:23.752164041 +00:00 client close; no transport to shut down  (from sentry)
14:01:27.420  
Compiled successfully.
```

**Comment by BYK** [comment]
Looks like an HTTP keep-alive issue. Are you running off of latest master of this repo, which has Nginx in front of Sentry now?

Also can you report back the values set in `SENTRY_WEB_OPTIONS` in your `sentry.conf.py` file?

**Comment by BYK** [comment]
Closing due to lack of response. That said I think this should be resolved by #576 as it fixes the errors with keep alive and when posting to uWSGI.
